### DISCUSSION TITLE
More precise time at the end of the game

### DISCUSSION THREAD

**Discussion by Prehistoricman** [ehmorris/lunar-lander#38] [Ideas]
I like to try to land as fast as possible so if it could say 6.x seconds rather than just 6 seconds that would be great. It would also add more competition for people to get lower times, like a speedrun.
I can make a PR if you like.
Thanks.

**Comment by ehmorris** [comment]
Hey @Prehistoricman sorry I missed this. Great idea. You'd need to change things here: https://github.com/ehmorris/lunar-lander/blob/master/lander/lander.js#L119C7-L121 and here: https://github.com/ehmorris/lunar-lander/blob/master/index.html#L51

Would love a PR. I can always implement it if you no longer have time.
